doc/API.md

```diff
@@ -737,12 +737,49 @@ returns active contracts. Expired contracts are not included.
 {
   "contracts": [
     {
-      "endheight":       50000, // block height
-      "id":              "1234567890abcdef0123456789abcdef0123456789abcdef0123456789abcdef",
-      "lasttransaction": {}, // types.Transaction
-      "netaddress":      "12.34.56.78:9",
-      "renterfunds":     "1234", // hastings
-      "size":            8192    // bytes
+      // Amount of contract funds that have been spent on downloads.
+      "downloadspending": "1234", // hastings
+
+      // Block height that the file contract ends on.
+      "endheight": 50000, // block height
+
+      // Fees paid in order to form the file contract.
+      "fees": "1234", // hastings
```

Comment:
@lukechampine: does this include (on the backend) the contract price paid to the host? Does that count as a fee?